### PR TITLE
fix docker-watch with graceful run and exit capability

### PR DESCRIPTION
**Pull request by chuckwired**
# Why
If your docker container exits unexpectedly, it can be difficult to debug what went wrong e.g. this issue: https://github.com/eddiesigner/liebling/issues/281. Add more graceful handling of log output to the `npm run docker-watch` command using `concurrently.js`.

https://www.npmjs.com/package/concurrently

# Screenshot
> Please note I accidentally called it `BROWSERSYNC` on my screenshot and personal branch, but the PR prefixes logs with `WEBPACK`.

Screenshot 2021-01-03 at 13 11 00

# Checklist
- [x] add concurrently dev dependency
- [x] update log outputs
- [x] ready for review/merge

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,5 +54,8 @@
     "desktop": "assets/screenshot-desktop.jpg",
     "mobile": "assets/screenshot-mobile.jpg"
   },
-  "version": "0.9.0"
+  "version": "0.9.0",
+  "devDependencies": {
```
Comment (eddiesigner) [inline package.json:58]: This `package.json` has metadata only relevant for Ghost, please move this dependency to the `package.json` file located in `src`.

**Review by eddiesigner — COMMENTED** [review]
This is really cool, thanks! 🙏 
I wrote a comment in one of the changes, hope you don't mind to solve it ;)

File: package.json (hunk 1 above)
Comment (chuckwired) [reply package.json:58]: Sure! Will update this tonight, and rebase :)

**Comment by chuckwired** [comment]
@eddiesigner updated and rebased, not sure if I should commit the `package-lock.json` file? If it's helpful, I'm using node `v15.5.1` (stable).

**Comment by eddiesigner** [comment]
@chuckwired Nope, that's just fine, I will merge it now, thanks!

**Pull request merged**